codefresh.yaml

```diff
@@ -4,32 +4,34 @@ stages:
 - Integration
 - Delivery
 
+
 steps:
 
   Prepare:
-    title: Update Helm package version  
+    title: Update Helm package version
     image: alpine:3.8
     stage: Integration
     commands:
     - apk add --no-cache curl jq python py-pip
     - pip install yq
     - sh ${{CF_VOLUME_PATH}}/iris/hack/.codefresh/push-helm.sh
+      # ^-- cfexports IRIS_VERSION from VERSION file
```

Comment:
`cf_export IRIS_VERSION=$(cat VERSION)` should do the job

Reply:
No, that's just a comment explaining that the script exports that.   It took me a while to find where `IRIS_VERSION` was coming from, so I added the comment pointing to that script.